core-primitives/stf-executor/src/traits.rs

```diff
@@ -31,28 +32,15 @@ pub enum StatePostProcessing {
 	Prune,
 }
 
-/// Execute shield funds on the STF
-pub trait StfExecuteShieldFunds {
-	fn execute_shield_funds(
-		&self,
-		account: AccountId,
-		amount: Amount,
-		shard: &ShardIdentifier,
-	) -> Result<H256>;
-}
```

Comment:
Removed, no longer needed. 'Shield funds' no longer is a special case for the STF executor, but just a regular trusted operation to be executed from the TOP pool.